src/tests/lib/components/Tooltip.spec.ts

```diff
@@ -16,15 +19,59 @@ describe("Tooltip", () => {
   });
 
   it("should render aria-describedby and relevant id", () => {
-    const { container } = render(TooltipTest, { text: "text" });
+    const { container } = render(TooltipTest, { text: "text", id });
     expect(
       container.querySelector("[aria-describedby='tid']"),
     ).toBeInTheDocument();
     expect(container.querySelector("[id='tid']")).toBeInTheDocument();
   });
 
+  it("should render different ID per tooltip when using idPrefix", () => {
+    render(TooltipTest, { text: "text", idPrefix });
+    const { container } = render(TooltipTest, { text: "text", idPrefix });
+    const tooltipTargets = container.querySelectorAll(".tooltip-target");
+    expect(tooltipTargets).toHaveLength(2);
+    const describedBy1 = tooltipTargets[0].getAttribute("aria-describedby");
+    const describedBy2 = tooltipTargets[1].getAttribute("aria-describedby");
+    expect(describedBy1).not.toEqual(describedBy2);
+    expect(describedBy1).toMatch(new RegExp(`^${idPrefix}-\\d+$`));
+    expect(describedBy2).toMatch(new RegExp(`^${idPrefix}-\\d+$`));
+    expect(
+      container.querySelector(`[id="${describedBy1}"]`),
+    ).toBeInTheDocument();
+    expect(
+      container.querySelector(`[id="${describedBy2}"]`),
+    ).toBeInTheDocument();
+  });
+
+  it("should render different ID per tooltip when not specifying any ID", () => {
```

Comment:
What is the difference between this test and the previous one?

I only see that in this one you don't check against the regex.

Reply:
Oops, this was supposed to not pass the `idPrefix` either. Fixed. Thanks.